tests/regression/tests/base_positive_rules.py

```diff
@@ -17,9 +17,7 @@
 skeletontest=0
 Meta='''---
   meta:
-    author: "spartantri"
-    enabled: true
-    name: "skeletonid.yaml"
+    author: "spartantri"    name: "skeletonid.yaml"
```

Comment:
We should probably remove the script. In the future, we should have some form of matrix / parameterization for go-ftw instead of hundrds of tests with small variations in the payload. https://github.com/coreruleset/go-ftw/issues/321